source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,218 @@
+---
+title: Direct Debit
+weight: 75
+---
+
+# GOV.UK Pay Variable Direct Debit
```

Comment:
Do we want to call this 'Variable Direct Debit' or just 'Direct Debit'? Either way one of the titles should change but I'm inclined to say just 'Direct Debit'? Reason being we talk about one-off payments fairly prominently.

Reply:
Direct Debit is better